homeassistant/components/switchbee/translations/en.json

```
@@ -0,0 +1,32 @@
+{
+  "config": {
+      "abort": {
+          "already_configured_device": "Device is already configured"
+      },
+      "error": {
+          "cannot_connect": "Failed to connect",
+          "invalid_auth": "Failed to Authenticate with the Central Unit",
+          "unknown": "Unexpected error"
+      },
+      "step": {
+          "user": {
+              "data": {
```

```suggestion
              "description": "Setup SwitchBee integration with Home Assistant.",
              "data": {
```
Description is missing:
https://github.com/home-assistant/core/blob/3d5fcbe634902ae6795b57784288d2f26b2b64a6/homeassistant/components/switchbee/strings.json#L5